addons/l10n_cz/data/tax_report.xml

```diff
@@ -0,0 +1,819 @@
+        <field name="country_id" ref="base.cz"/>
+        <field name="filter_hierarchy">optional</field>
+        <field name="filter_multi_company">disabled</field>
+        <field name="default_opening_date_filter">last_month</field>
+        <field name="filter_date_range" eval="True"/>
+        <field name="filter_period_comparison" eval="True"/>
+        <field name="filter_show_draft" eval="True"/>
+        <field name="column_ids">
+            <record id="l10n_cz_vat_declaration_tax_base" model="account.report.column">
+                <field name="name">Tax base</field>
+                <field name="expression_label">tax_base</field>
+                <field name="figure_type">monetary</field>
+            </record>
+            <record id="l10n_cz_vat_declaration_output_tax" model="account.report.column">
+                <field name="name">Output Tax</field>
+                <field name="expression_label">output_tax</field>
+                <field name="figure_type">monetary</field>
+            </record>
+            <record id="l10n_cz_vat_declaration_value" model="account.report.column">
+                <field name="name">Value</field>
+                <field name="expression_label">value</field>
+                <field name="figure_type">monetary</field>
+            </record>
+            <record id="l10n_cz_vat_declaration_in_full_amount" model="account.report.column">
+                <field name="name">In full</field>
+                <field name="expression_label">in_full_amount</field>
+                <field name="figure_type">monetary</field>
+            </record>
+            <record id="l10n_cz_vat_declaration_reduced_claim" model="account.report.column">
+                <field name="name">Short deduction</field>
+                <field name="expression_label">reduced_claim</field>
+                <field name="figure_type">monetary</field>
+            </record>
+            <record id="l10n_cz_vat_declaration_coefficient" model="account.report.column">
+                <field name="name">Coefficient</field>
+                <field name="expression_label">coefficient</field>
+                <field name="figure_type">percentage</field>
+            </record>
+        </field>
+        <field name="line_ids">
+            <record id="l10n_cz_vat_declaration_line_1" model="account.report.line">
+                <field name="name">I. Taxable transactions</field>
+                <field name="hierarchy_level">0</field>
+                <field name="children_ids">
+                    <record id="l10n_cz_vat_declaration_line_2" model="account.report.line">
+                        <field name="name">Supply of goods or services with a place of performance in the domestic territory</field>
+                        <field name="children_ids">
+                            <record id="l10n_cz_vat_declaration_line_3" model="account.report.line">
+                                <field name="name">Basic - line 1</field>
```

Comment:
Is "line 1" really part of the report?
Is it not risky to keep indexes like that in the code? If the report changes, we would need to change everything

Reply:
@william-andre This is the name indicated on the partner provided excel sheet https://docs.google.com/spreadsheets/d/1xqDqpLwY5nCyNA5WuzLSPdR52Jq6f8IPxU4S3Bh1Nw4/edit?pli=1&gid=1709389226#gid=1709389226

Reply:
I got that, but what is their source?
What does the official report look like?

Reply:
That is the source attached to the task https://adisspr.mfcr.cz/dpr/adis/idpr_epo/epo2/form/subjekt.faces
I don't understand most of it because it is in Czech, and the browser translation is awful 
Here is a screenshot from the lines (they have r.1 at the end)
![image](https://github.com/odoo/odoo/assets/160074741/4c9be19c-d16c-49ea-8993-f0a4128f5b56)